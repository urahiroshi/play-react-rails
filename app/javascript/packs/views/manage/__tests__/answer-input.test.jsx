import React from 'react'
import renderer from 'react-test-renderer'
import AnswerInput from '../answer-input'

describe('#render', () => {
  it('should show snapshot', () => {
    const component = renderer.create(
      <AnswerInput error="hoge" onChange={() => {}} />,
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
