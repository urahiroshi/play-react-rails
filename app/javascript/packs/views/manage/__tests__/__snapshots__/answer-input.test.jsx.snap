// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render should show snapshot 1`] = `
<div>
  <div>
    <input
      className="form-control manage-answer-input"
      defaultValue=""
      disabled={false}
      onChange={[Function]}
      placeholder="Answer"
      type="text"
    />
  </div>
  <div
    className="manage-error-message"
  >
    hoge
  </div>
</div>
`;
